### ISSUE TITLE
Removal of Command Icons Has No Effect

### ISSUE THREAD

**Issue by PaltryProgrammer** [hluk/CopyQ#1992]
Greetings Kind Regards I no doubt am doing something wrong but am too lazy to search further re/ removal of Command Icons. The attached GIF displays the current list of Command Icons in the Command Dialog post removal. However as can be seen upon update no change in displayed Command Icons occurred. 
![Removing Command Icons Has No Effect](https://user-images.githubusercontent.com/48535920/168484887-967d1685-a471-4067-95a3-c30dcf861e72.gif)
PS The black colored Command Icons are rather depressing. Some color would be appreciated. Perhaps I will edit the icons or load others when time permits.

**Comment by hluk** [comment]
Yeah, the behavior is still a bit awkward - you have to **clear the check marks instead** instead of removing the commands completely otherwise they will be added again.

> PS The black colored Command Icons are rather depressing. Some color would be appreciated. Perhaps I will edit the icons or load others when time permits.

You can select the app theme in Appearance config tab in Preferences dialog and also enable "Set colors for tabs, tool bar and menus" option there so the whole main window style changes. Though this would set same color for all icons.
